htsget.md

```diff
@@ -216,6 +216,97 @@ QUAL	| Base quality scores
 
 Example: `fields=QNAME,FLAG,POS`.
 
+
+# Method: get variants by ID
+
+    GET /variants/<id>
+
+The core mechanic for accessing specified variant data. The JSON response is a "ticket" similar to `reads` method.
+
+The client can request only variants overlapping a given genomic range. The response may however contain a superset of the desired results, including all records overlapping the range, and potentially other records not overlapping the range; the client should filter out such extraneous records if necessary. Successful requests with empty result sets still produce a valid response in the requested format (e.g. including header and EOF marker).
+
+<table>
+<tr markdown="block"><td>
+`id`  
+_required_
+</td><td>
+Study ids from which variants are to be returned.
```

Comment:
Should this be just 'Ids' instead of 'Study IDs' (For EGA this will initially have to be File IDs)

Reply:
Also: Would it be sufficient to use a single ID (in line with BAM/CRAM specs), and wait for a unified way to submit multiple ID via POST?

Reply:
I would be fine with changing it to "one or multiple IDs". Using only one wouldn't be sufficient for the EVA use case for instance as we support cross-study queries.

It could be useful to list examples of possible types of identifiers (studies, files, samples) for both reads and variants endpoints, just to make clear what could be supported. I helped to define this addition and we were a bit unsure at first.

Reply:
On the last call, we agreed that we would add POST support to support mulitple IDs. Otherwise, the url could/will get truncated unless there is a hard bound on the total number of characters (maybe http already defines this).

Reply:
This seems orthogonal to VCF support --- perhaps we should limit to using GET with a single ID here for now, and tackle multiple IDs/POST for both reads and variants in a separate PR (as @AlexanderSenf  suggests)?